api/URLSearchParams.json

```
@@ -173,6 +217,57 @@
               "deprecated": false
             }
           }
+        },
+        "sequence": {
+          "__compat": {
+            "description": "sequence for <code>init</code> object",
+            "support": {
+              "chrome": {
+                "version_added": "58"
+              },
+              "chrome_android": {
+                "version_added": "58"
+              },
+              "edge": {
+                "version_added": "17"
+              },
+              "firefox": {
+                "version_added": "53"
+              },
+              "firefox_android": {
+                "version_added": "53"
+              },
+              "ie": {
+                "version_added": false
+              },
+              "nodejs": {
+                "version_added": "7.10.0"
+              },
+              "opera": {
+                "version_added": "45"
+              },
+              "opera_android": {
+                "version_added": "45"
```

```suggestion
                "version_added": "43"
```

I think Opera Android 43 (with Blink 59) is the first release with support.

Opera Android 42 has Blink 55

---

Good catch! This squares with https://github.com/mdn/browser-compat-data/blob/master/browsers/opera_android.json#L206

I'm going to apply this, approve, and merge. 👍